plugins/tracker-resources/src/components/issues/edit/SubIssues.svelte

```diff
@@ -72,6 +73,14 @@
   $: viewOptions = viewlet !== undefined ? getViewOptions(viewlet, $viewOptionStore) : undefined
 
   export let focusIndex = -1
+
+  let lastIssueId: Ref<Issue>
+  afterUpdate(() => {
+    if (lastIssueId !== issue._id) {
+      lastIssueId = issue._id
+      isCreating = $draftsStore[`${issue._id}_subIssue`] !== undefined
```

Comment:
I suppose we have some constants in drafts? Let's use them and not hardcode.

Reply:
No, here key generated from parent issue id